### PR TITLE
workload: Allow my_vars dir location and minor SSH ansible fix

### PR DESCRIPTION
**Pull request by fbladilo**
This PR does two things : 

- Allow the optional setting (-m) of my_vars.yml directory location, the default continues to be ../<OCP_VERSION>.x

This helps the Jenkins use case when there are multiple cluster deployments coming from the same mig-agnosticd cloned repo , i.e "4.1", "nightly"

- Set default ansible SSH options to turn off strict checking. This is needed in order to fully automate these workloads without interactive prompts for host checks during playbook runs.

@pranavgaikwad @jwmatthews

### CODE REVIEW

File: workloads/workload.yml [hunk 1]
```diff
@@ -42,8 +42,8 @@
   - name: "Including cluster variables..."
     include_vars: "{{ item }}"
     loop:
-      - "../{{ ocp_version | mandatory }}.x/my_vars.yml"
-      - "../{{ ocp_version }}.x/ocp{{ ocp_version }}_vars.yml"
+      - "{{ my_vars_dir | mandatory }}/my_vars.yml"
```
Comment (pranavgaikwad) [inline workloads/workload.yml]: @fbladilo do you want to make this var mandatory?

Comment (fbladilo) [reply workloads/workload.yml]: @pranavgaikwad It has to be set, the same variable now contains the default location for my_vars.yml directory (../<OCP_VERSION>.x) or whaterver is optionally set by "-m".

Comment (pranavgaikwad) [reply workloads/workload.yml]: understood

**Comment by pranavgaikwad** [comment]
LGTM

**Pull request merged**